experimental/rust-tide/README.md

```diff
@@ -0,0 +1,86 @@
+# Rust Tide Stack
+
+The Rust Tide stack provides a consistent way of developing [tide](https://github.com/http-rs/tide) http servers.
+
+Designed to be used with [Appsody](https://appsody.dev/) an [open source](https://github.com/appsody/) development and operations accelerator for containers.
+
+This stack is based on the `Rust 1.42` runtime.
+
+## Templates
+
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the default template project. This template provides a http server that returns "Hello World" on http://localhost:8000/hello.
+
+## Getting Started
+
+1. Clone this repo and configure appsody
+   ```bash
+   git clone https://github.com/No9/rust-tide
+   cd rust-tide
+   appsody stack package
+   ```
+
+1. Create a new folder in your local directory and initialize it using the Appsody CLI, e.g.:
+
+    ```bash
+    mkdir my-project
+    cd my-project
+    appsody init dev.local/rust-tide
+    ```
+    This will initialize a Tide project using the default template.
+
+1. After your project has been initialized you can then run your application using the following command:
+
+    ```bash
+    appsody run
+    ```
+
+    This launches a Docker container that continuously re-builds and re-runs your project. It also exposes port 8000 to allow you to call your service from your browser and test tooling.
+
+1. You should see a message printed on the console:
+
+    ```Running `server/bin/target/debug/rust-tide-server```
```

Comment:
Is there any way we can indicate to the user that they need to go to `localhost:8000` via the terminal?